dev/tasks/python-wheels/github.osx.amd64.yml

```diff
@@ -35,7 +35,7 @@ env:
 jobs:
   build:
     name: Build wheel for OS X
-    runs-on: macos-10.15
+    runs-on: macos-12
```

Comment:
Can we keep this at 11 for now?

Reply:
Yes.
I changed it to `macos-latest` that equals to `macos-11` for now.

I tried `macos-12` to resolve the crc32c related failure. But it's not unrelated. So we don't need to use `macos-12` here.